content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Kubernetes operators
+reviewers:
+- erictune
+- lavalamp
+- cheftako
+- chenopis
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Operators are software extensions to Kubernetes that make use of third-party
+resources to manage applications and their components. Operators follow
+Kubernetes priciples, notably the [control loop](/docs/concepts/#kubernetes-control-plane).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Motivation
+
+The Operator concept aims to capture the key aim of a human operator who
+is managing a service or set of services. Human operators who look after
+specific applications and services have deep knowledge of how the system
```

Comment:
Is software `pattern` a better description than `concept`?
See line 125 in the previous file. The content in this file should tie in with the content in, https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster